v3.1/custom-rules/before-crs.dist/cleanlogs.conf

```diff
@@ -8,3 +8,8 @@
 # - allow request
 # - don't write an audit log for the request
 SecRule REMOTE_ADDR "^10\.1\.\d+\.1$" "phase:1,id:40000,nolog,allow,ctl:auditEngine=Off"
+
+# Ignore requests coming from localhost. This is useful if there are request issued via Shell or
+# from a sidecar that provide limited or no customization in the request headers (e.g. Apache exporter).
+SecRule REMOTE_ADDR "@ipMatch 127.0.0.1" "phase:1,id:40001,nolog,pass,t:none,ctl:auditEngine=Off"
+SecRule REMOTE_ADDR "@ipMatch ::1" "phase:1,id:40002,nolog,pass,t:none,ctl:auditEngine=Off"
```

Comment:
I think, we can replace `pass,t:none` by `allow` (as in the rule for the health checks above).

- `t:none` ... transformation = none, is probably the default
- `pass` ... tells ModSecurity to continue processing the next rule(s); `allow`, on the other hand, stops evaluating any further rules for this request, which saves some CPU cycles

Reply:
Thanks for clarification. updated it.